mne/epochs.py

```diff
@@ -245,6 +245,15 @@ class BaseEpochs(ProjMixin, ContainsMixin, UpdateChannelsMixin,
         See :class:`mne.Epochs` docstring.
 
         .. versionadded:: 0.16
+    event_repeated : str
+        How to handle duplicate time samples with different event codes.
+        Can be 'error' (default), to raise an error, 'drop' to only retain the
+        row occurring dirst in the events array, or 'merge' to create a new
+        event code that reflects a co-occurrence of several events at the same
+        time. For the 'merge' option, the event_id dictionary will be updated
+        accordingly.
```

Comment:
```suggestion
        accordingly (see Notes for details).
```
And in Notes say that what is done is 
```
            event_id['{event_id_1}/{event_id_2}/...'] = new_id_number
```
or so